CONTRIBUTING.md

```diff
@@ -610,6 +610,32 @@ Then put this:
 export NODE_OPTIONS=--no-experimental-fetch
 ```
 
+If Following type of error comes while running dev server(i.e using above commands):
+
+```bash
+Error: ENOSPC: System limit for number of file watchers reached
+```
+The error is thrown because the number of files monitored by the system has reached the limit.
+The error can be fixed by increasing the amount of inotify watchers.
+
+Do the following to Modify the number of system monitoring files :
+
+
+The current value of max watches can be checked by below command.
```

Comment:
```suggestion
The current value of max watches can be checked with:
```

Reply:
This PR sometimes says "watches" and sometimes "watchers" - is it correct that both terms get used in this context?